oss-internship-2020/libtiff/example/main_sandboxed.cc

```diff
@@ -0,0 +1,277 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstring>
+#include <iostream>
+#include <vector>
+
+#include "../sandboxed.h"
+#include "sandboxed_api/sandbox2/util/fileops.h"
+#include "sandboxed_api/sandbox2/util/path.h"
+#include "tiffio.h"
+
+/* sapi functions:
+ *    TIFFTileSize
+ *    TIFFOpen
+ *    TIFFReadEncodedTile
+ *    TIFFSetField
+ *    TIFFClose
+ *    TIFFReadRGBATile
+ *    TIFFGetField
+ */
+
+static const std::vector<unsigned char> cluster_0 = {0, 0, 2, 0, 138, 139};
+static const std::vector<unsigned char> cluster_64 = {0, 0, 9, 6, 134, 119};
+static const std::vector<unsigned char> cluster_128 = {44, 40, 63, 59, 230, 95};
+
+static int CheckCluster(int cluster,
+                        const sapi::v::Array<unsigned char> &buffer,
```

Comment:
Prefer to keep `*` and `&` close to the type. SAPI's own `.clang-format` file now enforces this. Your style is generally allowed, too, but for consistency we prefer asterisks and ampersands to the left.